embedded-graphics/src/image/image_bmp.rs

```diff
@@ -121,10 +122,33 @@ where
     }
 }
 
-impl<'a, C> Drawable for ImageBmp<'a, C> where C: PixelColor + From<<C as PixelColor>::Raw> {}
+impl<'a, 'b, C> IntoIterator for &'b ImageBmp<'a, C>
+where
+    'b: 'a,
+    C: PixelColor + From<<C as PixelColor>::Raw>,
+{
+    type Item = Pixel<C>;
+    type IntoIter = ImageBmpIterator<'a, C>;
+
+    // NOTE: `self` is a reference already, no copies here!
```

Comment:
Can you lose this copypasta? I'm sick of seeing my comment everywhere :joy:

Reply:
lol, that is embarrassing! Totally missed that. Turns out the copy-pasta isn't even necessary now so I'm just going to remove that whole impl for &mut.